.circleci/config.yml

```
@@ -11,7 +11,9 @@ commands:
       - run:
           name: Install Node.js
           command: |
-            curl -fsSL https://deb.nodesource.com/setup_lts.x | sudo -E bash -
+            curl -SLO https://deb.nodesource.com/nsolid_setup_deb.sh
+            chmod 500 nsolid_setup_deb.sh
+            ./nsolid_setup_deb.sh 18
```

Probably requires root?

```suggestion
            sudo ./nsolid_setup_deb.sh 18
```